Sources/Apollo/MultipartFormData.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// A helper for building out multi-part form data for upload
-class MultipartFormData {
+public class MultipartFormData {
```

Comment:
1) Why does this need to be public? 
2) If it needs to be public, why aren't any other methods or properties public?

Reply:
1: Because the compiler is not happy otherwise :(
2: The actually should. If anyone make their own multipart creator they should be accessible

Reply:
Pushed public methods.